diff --git a/src/utils/auditLog.js b/src/utils/auditLog.js
index 8ab3663cb44fb2d9b18c68253933c8c8afaab9ea..d19adc76a935f8e06616b2a272307f4b5ce98a59 100644
--- a/src/utils/auditLog.js
+++ b/src/utils/auditLog.js
@@ -1,6 +1,9 @@
 // Función de registro de auditoría (temporal para evitar error)
-const createAuditLog = async (action, userId) => {
-  console.log(`Audit log - Acción: ${action}, Usuario: ${userId}`);
+// Ahora recibe un único objeto con las propiedades { accion, usuario, descripcion, ip }
+const createAuditLog = async ({ accion, usuario, descripcion, ip }) => {
+  console.log(
+    `Audit log - Acción: ${accion}, Usuario: ${usuario}, Descripción: ${descripcion}, IP: ${ip}`
+  );
 };
 
 module.exports = { createAuditLog };
